libs/dh/core/shell/src/lib/dh-core-shell.component.ts

```diff
@@ -37,10 +39,13 @@ import { DhPrimaryNavigationComponent } from './dh-primary-navigation.component'
   selector: 'dh-shell',
   styleUrls: ['./dh-core-shell.component.scss'],
   templateUrl: './dh-core-shell.component.html',
+  providers: [DhBreadcrumbService],
```

Comment:
issue: the service is provided in `root` so there's no need to provide it here as well.

Reply:
Fixed